mkosi/backend.py

```diff
@@ -752,6 +752,7 @@ def run(
     delay_interrupt: bool = True,
     stdout: _FILE = None,
     stderr: _FILE = None,
+    env: Mapping[str, Any] = {},
```

Comment:
In principle this is not problematic, since this dict is not changed in the body of the function, but it is a potential waiting footgun, since (at least until late-bound arguments land) this is only initialised to the empty dict once and will be the same dict whenever `run` is invoked.

Reply:
Yeah, but I figure the typing gives sufficient protection against these type of footguns. I do agree that if the type was Dict, we wouldn't want to do this